src/ucp/wireup/address.c

```diff
@@ -44,6 +44,13 @@ typedef struct {
 } ucp_address_packed_device_t;
 
 
+typedef struct {
+    float            overhead;
+    float            bandwidth;
+    uint32_t         prio_cap_flags; /* 8 lsb: prio, 24 msb - cap flags */
+} ucp_address_packed_iface_attr_t;
+
+
 #define UCP_ADDRESS_FLAG_LAST         0x80   /* Last address in the list */
 #define UCP_ADDRESS_FLAG_EMPTY        0x80   /* Device without TL addresses */
```

Comment:
they are used in different parts in the packed address and i didn't want to force them to be the same